packages/gatsby-plugin-sharp/src/utils.js

```diff
@@ -88,8 +88,27 @@ export function rgbToHex(red, green, blue) {
     .slice(1)}`
 }
 
+const checkIgnoredParameters = (layout, parameters, filepath) => {
+  const ignoredParams = Object.entries(parameters).filter(([_, value]) =>
+    Boolean(value)
+  )
+  if (ignoredParams.length) {
+    console.warn(
```

Comment:
Please use the reporter
```suggestion
    console.warn(
```